packages/financial-templates-lib/test/clients/FinancialContractClient.js

```diff
@@ -824,23 +831,26 @@ contract("FinancialContractClient.js", function(accounts) {
 
             // Set a funding rate
             await _setFundingRateAndAdvanceTime(toWei("0.000005"));
-            await financialContract.applyFundingRate();
+            // await financialContract.applyFundingRate();
 
-            // funding rate should be set within contract.
-            assert.equal((await financialContract.fundingRate()).cumulativeMultiplier.toString(), toWei("1.05"));
+            // // funding rate should be set within contract.
+            // assert.equal((await financialContract.fundingRate()).cumulativeMultiplier.toString(), toWei("1.05"));
 
-            // funding rate is not applied until the client is updated.
-            assert.equal(client.getLatestCumulativeFundingRateMultiplier(), toWei("1"));
+            // // funding rate is not applied until the client is updated.
+            // assert.equal(client.getLatestCumulativeFundingRateMultiplier(), toWei("1"));
 
-            // After updating the client the funding rate is applied.
-            await client.update();
-            assert.equal(client.getLatestCumulativeFundingRateMultiplier(), toWei("1.05"));
+            // // After updating the client the funding rate is applied.
+            // await client.update();
+            // assert.equal(client.getLatestCumulativeFundingRateMultiplier(), toWei("1.05"));
 
             // after advancing time with the same funding rate the client value should not change
             await _setFundingRateAndAdvanceTime(toWei("0"));
-            await financialContract.applyFundingRate();
+            // await financialContract.applyFundingRate();
             assert.equal(client.getLatestCumulativeFundingRateMultiplier().toString(), toWei("1.05"));
```

Comment:
I've commented out the explicit `applyFundingRate()` calls here, and therefore the client should be able to detect the pending funding rate if the multicontract simulation is working